src/main/kotlin/com/atlassian/performance/tools/sshubuntu/api/SshUbuntuContainer.kt

```diff
@@ -17,13 +17,15 @@ class SshUbuntuContainer(
     }
 
     fun start(): SshUbuntu {
-        val version = SshUbuntuProperties().version
```

Comment:
I'm not sure if this `SshUbuntuProperties` and it's `app.properties` loaded from JAR resources shouldn't be treated as API. It's kinda global configuration mechanism of this class.

Might be worth checking what did we promise in the CHANGELOG.

Reply:
On the other side we may not be breaking API by not using the values from `app.properties` anymore. I'm not sure.

Reply:
I haven't found any other usage of app.properties other than extracting the version.
Its purpose was to pull the same image from docker repository as library version